### ISSUE TITLE
[0.99.13 Linux] Unhandled exception & subsequent crash when pasting '-sU' 

### ISSUE THREAD

**Issue by Marnick39** [giuspen/cherrytree#1200]
When pasting '-sU' in in a rich text (sub)tree crashes the application with the following message:

```
unhandled exception (type std::exception) in signal handler:
what: basic_string::at: __n (which is 3) >= this->size() (which is 3)
fish: “cherrytree” terminated by signal SIGTRAP (Trace or breakpoint trap)

```
I am able to replicate this crash over and over again, in clean instances:*
![Peek 2020-10-01 22-04](https://user-images.githubusercontent.com/17315511/94857619-5582cf00-0421-11eb-9609-8cc113fb977a.gif)

*Somehow copying '-sU' from outside the application is necessary to trigger this bug

---
Full cherrytree logs from bug triggering test run:
```
[2020-10-01 22:08:34.167] [   ] [debug] /home/XXXXX/.config/cherrytree/config.cfg parsed
[2020-10-01 22:08:59.574] [   ] [debug] Node name: Device Node
[2020-10-01 22:09:58.524] [   ] [debug] Node name: test
[2020-10-01 22:10:14.906] [gtk] [info] 
unhandled exception (type std::exception) in signal handler:
what: basic_string::at: __n (which is 3) >= this->size() (which is 3)
fish: “cherrytree” terminated by signal SIGTRAP (Trace or breakpoint trap)
```
If you need any more info, I'll be happy to provide it!

**Comment by txe** [comment]
Hi, I believe, it was fixed in 0.99.14

**Comment by Marnick39** [comment]
Thanks for the quick reply!
I've got my cherrytree package from the AUR, which carries the outdated label as of today. I'll be checking 0.99.14 and will report back as soon as it gets pushed out.

**Comment by txe** [comment]
Sure, thanks

**Comment by kettenfett** [comment]
#1186

**Comment by giuspen** [comment]
this was definitely fixed in 0.99.14, I'll close for now
